src/components/ScoreSetHistogram.vue

```diff
@@ -1,17 +1,17 @@
 <template>
   <TabMenu class="mave-histogram-viz-select" v-if="hasTabBar" v-model:activeIndex="activeViz" :model="vizOptions" />
-  <div v-if="hasTabBar && showShaders" class="mave-histogram-controls">
+  <div v-if="!hasTabBar && showShaders" class="mave-histogram-controls">
```

Comment:
I believe this should remain as `hasTabBar`, otherwise we might show ranges when clinical features are disabled. I think the weirdness with this probably stems from line 449. I'm not sure why we would not want to display the title if the tab bar exists. So I think that line can probably change to `.title('Distribution of Functional Scores')`.

Reply:
I agree. I'll set it back to `hasTabBar` and modify the title part. Maybe it was a typo?